stories/molecules/Teaser.stories.tsx

```diff
@@ -20,14 +20,98 @@ export const WithAllFields: Story = {
         title: titleText,
         subtitle: 'Awsome inc.',
         hasCheckbox: true,
-        timestamp: 'Today',
+        date: 'Today',
         primaryInfo: { text: 'Primary', href: 'https://textkernel.com' },
         secondaryInfo: { text: 'Secondary' },
         tercearyInfo: { text: 'Terceary' },
         sourceInfo: { text: 'Source', icon: <FaLinkedinIn /> },
         isVisited: true,
         matchingIndicatorPercentage: 50,
-        tags: [<Tag bgColor="yellow">First</Tag>, <Tag bgColor="lightblue">Second</Tag>],
+        bottom: (
+            <div style={{ display: 'flex', gap: '8px' }}>
+                <Tag bgColor="yellow" size="small">
+                    First
+                </Tag>
+                <Tag bgColor="lightblue" size="small">
+                    Second
+                </Tag>
+            </div>
+        ),
+    },
+    render: (args) => {
+        const [isSelected, setIsSelected] = React.useState(false);
+
+        return (
+            <Teaser {...args} isSelected={isSelected} onChange={() => setIsSelected(!isSelected)} />
+        );
+    },
+};
+
+export const VariantOne: Story = {
+    name: 'Variant One',
+    args: {
+        title: 'John Doe',
+        subtitle: 'Surgeon',
+        hasCheckbox: true,
+        primaryInfo: { text: 'Netherlands' },
+        secondaryInfo: { text: 'example@example.com', href: 'mailto:example@example.com' },
+        tercearyInfo: { text: '+99 989 989 989', href: 'tel:+99989989989' },
+        isVisited: true,
+        matchingIndicatorPercentage: 25,
+    },
+    render: (args) => {
+        const [isSelected, setIsSelected] = React.useState(false);
+
+        return (
+            <Teaser {...args} isSelected={isSelected} onChange={() => setIsSelected(!isSelected)} />
+        );
+    },
+};
+
+export const VariantTwo: Story = {
+    name: 'Variant Two',
+    args: {
+        title: 'John Doe',
+        subtitle: 'Surgeon',
+        hasCheckbox: true,
+        primaryInfo: { text: 'Netherlands' },
+        secondaryInfo: { text: 'example@example.com', href: 'mailto:example@example.com' },
+        tercearyInfo: { text: '+99 989 989 989', href: 'tel:+99989989989' },
+        sourceInfo: {
+            text: 'Internet',
+            icon: <FaInternetExplorer />,
+        },
+        isVisited: true,
+    },
+    render: (args) => {
+        const [isSelected, setIsSelected] = React.useState(false);
+
+        return (
+            <Teaser {...args} isSelected={isSelected} onChange={() => setIsSelected(!isSelected)} />
+        );
+    },
+};
+
+export const VariantThree: Story = {
```

Comment:
It would be good to see a realistic example for JD. Sorry I still have difficulty seeing how it will look like. Could you render it in a div with some given width (500px?)? Can you use some job title (maybe even a long one to see how it will be truncated); company name, show multiple sources like `jobboard.com (+ 4 more)`, longer time indicator like `about 1 month ago`; no visitation indication. Maybe at least one variant could also loose the checkbox.